libs/server/Storage/Session/ObjectStore/SortedSetOps.cs

```diff
@@ -1126,57 +1126,72 @@ private GarnetStatus SortedSetUnion<TObjectContext>(ReadOnlySpan<ArgSlice> keys,
 
             // Get the first sorted set
             var status = GET(keys[0].ToArray(), out var firstObj, ref objectContext);
+
+            if (status == GarnetStatus.WRONGTYPE)
+            {
+                return GarnetStatus.WRONGTYPE;
+            }
+
+            Dictionary<byte[], double> sortedSetDictionary = null;
+            if (status == GarnetStatus.NOTFOUND)
+            {
+                sortedSetDictionary = new Dictionary<byte[], double>(ByteArrayComparer.Instance);
```

Comment:
Instead of creating an empty dictionary here and immediately cloning it later, can we directly create `pairs` as empty later on if `sortedSetDictionary` is null. That way we avoid the extra object creation here.

Reply:
Done